bootstrap-themes/pom.xml

```diff
@@ -40,16 +39,21 @@
         </dependency>
 
         <dependency>
-            <groupId>org.webjars</groupId>
+            <groupId>org.webjars.npm</groupId>
             <artifactId>bootstrap-italia</artifactId>
             <version>${bootstrap-italia.version}</version>
+            <exclusions>
+                <exclusion>  <!-- exclude dependency, because causing dependency conflict -->
```

Comment:
What kind of conflict does it cause ?

Reply:
Output : 

Failed to execute goal on project wicket-bootstrap-themes: Could not resolve dependencies for project de.agilecoders.wicket:wicket-bootstrap-themes:bundle:7.0.7-SNAPSHOT: Failed to collect dependencies for de.agilecoders.wicket:wicket-bootstrap-themes:jar:7.0.7-SNAPSHOT: Could not resolve version conflict among [org.webjars.npm:bootstrap-italia:jar:2.8.7 -> org.webjars.npm:video.js:jar:[8.0.4,9.0.0-0) -> org.webjars.npm:m3u8-parser:jar:[7.1.0,8.0.0-0) -> org.webjars.npm:videojs__vhs-utils:jar:[3.0.5,4), org.webjars.npm:bootstrap-italia:jar:2.8.7 -> org.webjars.npm:video.js:jar:[8.0.4,9.0.0-0) -> org.webjars.npm:mpd-parser:jar:[1.2.2,2.0.0-0) -> org.webjars.npm:videojs__vhs-utils:jar:[4.0.0,5.0.0-0), org.webjars.npm:bootstrap-italia:jar:2.8.7 -> org.webjars.npm:video.js:jar:[8.0.4,9.0.0-0) -> org.webjars.npm:videojs__vhs-utils:jar:[4.0.0,5.0.0-0), org.webjars.npm:bootstrap-italia:jar:2.8.7 -> org.webjars.npm:video.js:jar:[8.0.4,9.0.0-0) -> org.webjars.npm:videojs__http-streaming:jar:3.13.1 -> org.webjars.npm:videojs__vhs-utils:jar:4.0.0, org.webjars.npm:bootstrap-italia:jar:2.8.7 -> org.webjars.npm:video.js:jar:[8.0.4,9.0.0-0) -> org.webjars.npm:aes-decrypter:jar:[4.0.1,5.0.0-0) -> org.webjars.npm:videojs__vhs-utils:jar:[3.0.5,4)] -> [Help 1]